app/services/check/two-part.service.js

```diff
@@ -108,6 +111,10 @@ async function _fetchAndApplyReturns (billingPeriod, chargeVersion) {
         .where('endDate', '>=', billingPeriod.startDate)
         .whereJsonPath('metadata', '$.isTwoPartTariff', '=', true)
         .where(ref('metadata:purposes[0].tertiary.code').castInt(), legacyId)
+
+      chargePurpose.returnStatus = chargePurpose.returns.map((matchedReturn) => {
```

Comment:
I have used the name `matchedReturn` as I can't use `return` for obvious reasons. Shout if you have a better solution.